CHANGELOG.md

```diff
@@ -31,6 +31,9 @@ These changes are available on the `master` branch, but have not yet been releas
 
 ### Fixed
 
+- Fixed a bug in `Page.update_files` where `io.BytesIO` files didn't send properly more
+  than once. ([#1881](https://github.com/Pycord-Development/pycord/pull/1881))
+
 - Fixed a bug in `Page.update_files` where file objects stored in memory were causing an
   `AttributeError`. ([#1869](https://github.com/Pycord-Development/pycord/pull/1869))
```

Comment:
Can we combine these audit log entries?

Reply:
Not at my desk rn, please do as you see fit.

Reply:
This should be sufficient:
- Fixed bugs in `Page.update_files` where file objects stored in memory were causing an
  `AttributeError` and `io.BytesIO` files didn't send properly more
  than once. ([#1869](https://github.com/Pycord-Development/pycord/pull/1869) and [#1881](https://github.com/Pycord-Development/pycord/pull/1881))
```suggestion
- Fixed bugs in `Page.update_files` where file objects stored in memory were causing an
  `AttributeError`, and `io.BytesIO` files didn't send properly more
  than once. ([#1869](https://github.com/Pycord-Development/pycord/pull/1869) and [#1881](https://github.com/Pycord-Development/pycord/pull/1881))
```